dev/attest/src/testTsVersions.ts

```diff
@@ -0,0 +1,28 @@
+import { rmSync, writeFileSync } from "node:fs"
+import { join } from "node:path"
+import { exit } from "node:process"
+import { fromPackageRoot, readPackageJson } from "../../runtime/fs.js"
+import { shell } from "../../runtime/shell.js"
+;(() => {
```

Comment:
Any reason that this is embedded in an arrow function? Looks like it would be equivalent to just moving it to the top level of the file.